### ISSUE TITLE
Random number generation slow in Docker

### ISSUE THREAD

**Issue by aro** [web-push-libs/webpush-java#40]
We are using this library on an app that runs in a Docker instance. We noticed that the random number generation used for the salt can be very slow and block the execution for up to 5 minutes, just to generate the 16 bytes. 

The problem is with this line in PushService:
```java
byte[] salt = SecureRandom.getSeed( 16 );
```

Is there any specific reason to use `getSeed()` ?

Why not use nextBytes() instead? This random number generation is not blocking:
```java
byte[] salt = new byte[16];
new SecureRandom().nextBytes( salt );
```

I can create a pull request for this if it helps.

**Comment by GlennRicaud** [comment]
+1

**Comment by MartijnDwars** [comment]
Thanks for opening an issue. We've seen this issue before and the [FAQ in the README](https://github.com/web-push-libs/webpush-java#why-does-encryption-take-multiple-seconds) provides some solutions. Did these solutions not work for you?

> Why not use nextBytes() instead? This random number generation is not blocking:

The difference here is that `getSeed` gets entropy from a true random seed, whereas `getBytes(salt)` gets pseudorandom bytes based on some pre-computed seed. I'm not sure if there are security implications to using a PRNG (`nextBytes`) to generate the salt instead of a true random source (`getSeed`).

**Comment by martijndwars** [comment]
Thanks for opening the issue. I've removed the comment I placed earlier today because it contained wrong assumptions. I was under the impression that `SecureRandom.getSeed` would be more secure, but I doubt this is actually the case. On top of that, using `SecureRandom.getSeed` is discouraged according to the API docs (it's only there for backwards compatibility).

Switching to `secureRandom.nextBytes` sounds like a good plan. If you can create a PR then I'd be happy to merge and publish a new version to Maven Central. I think many users will appreciate this fix, since we've had this issue before (see [here](https://github.com/web-push-libs/webpush-java/issues/6) and [here](https://github.com/web-push-libs/webpush-java#why-does-encryption-take-multiple-seconds)).

**Comment by aro** [comment]
I created a [pull request](https://github.com/web-push-libs/webpush-java/pull/41).

Thanks for the fast response. And for the project, it's the best Web Push library in Java.
Bedankt 👍

**Comment by martijndwars** [comment]
Thanks! I've merged the PR and released version 3.0.1.
